packages/heartwood-components/components/03-structure/page.scss

```diff
@@ -110,6 +110,7 @@
 }
 
 .page__header-title-wrapper {
+	align-items: center;
```

Comment:
This needs to be wrapped in a media query so that it stacks on mobile and goes inline on larger screens. I think it's safe to use our `small` breakpoint for this.

![Screen Shot 2019-07-11 at 1 17 23 PM](https://user-images.githubusercontent.com/13734175/61078726-7efba900-a3de-11e9-86c5-140a3917fcd5.png)

Reply:
Updated 👍